### ISSUE TITLE
Year axis truncated in monthly edit view

### ISSUE THREAD

**Issue by gwern** [x-tools/SuperCount#22]
When viewing my edits by month, the year label on the left for each month is truncated eg. in https://tools.wmflabs.org/supercount/index.php?user=gwern&project=en.wikipedia.org&toplimit=10 I see '6/09', not '2006/09', and likewise on the bottom, '4/08', not '2014/08'.

Here is an Iceweasel screenshot: https://imgur.com/C4AtPeV

**Comment by cyberpower678** [comment]
Have you tried a different browser?  I know it works in common browsers.  I've never heard of Iceweasel until today.

**Comment by gwern** [comment]
Iceweasel is the Debian version of Firefox.

**Comment by cyberpower678** [comment]
It doesn't look like your browser is handling the SVG correctly.  The font looks too large for the image, try switching to a different browser and tell me if that resolves the issue.

Gesendet von Maximilian's iPhone. 
(Sent from Maximilian's iPhone.)

**Comment by gwern** [comment]
No, there's still truncation when I view the same URL in Epiphany, so it's not just a Firefox/Iceweasel issue: https://i.imgur.com/rSOltOC.png

**Comment by cyberpower678** [comment]
I've never heard of that browser either.

Gesendet von Maximilian's iPad. 
(Sent from Maximilian's iPad.)

**Comment by gwern** [comment]
I've demonstrated the problem on two browsers using entirely separate rendering engines (Firefox and Webkit), so just because you haven't heard of them doesn't mean your HTML/CSS/SVG is perfect...

**Comment by cyberpower678** [comment]
I never said that it was, otherwise I would have arrogantly closed this issue.  It's something I will indeed have to look into since browser compatibility is one of the most parts of designing this tool to me. 

Gesendet von Maximilian's iPad. 
(Sent from Maximilian's iPad.)

**Comment by TParis** [comment]
Relax Gwern.  Cyberpower678 is a volunteer and has provided these tools as a service and they are pretty damn impressive tools.  If you don’t like the HTML/CSS/SVG and you can’t express your concerns politely without snapping at a volunteer who is trying to help you, then use another tool.

v/r,

TP
